contrib/endpoints/src/api_manager/request_handler.cc

```diff
@@ -30,7 +30,24 @@ namespace google {
 namespace api_manager {
 
 void RequestHandler::Check(std::function<void(Status status)> continuation) {
-  auto interception = [continuation, this](Status status) {
+  if (context_) {
```

Comment:
The logic here should be:
check loadiing_status,  if not done,  registered a callback to do CreateRequestContext and InternalCheck().